zebra-consensus/src/transaction.rs

```diff
@@ -611,24 +678,18 @@ where
             // feed all of the inputs to the script verifier
             // the script_verifier also checks transparent sighashes, using its own implementation
             let inputs = transaction.inputs();
-            let known_utxos = request.known_utxos();
             let upgrade = request.upgrade(network);
 
             let script_checks = (0..inputs.len())
                 .into_iter()
                 .map(move |input_index| {
-                    let utxo_sender = utxo_sender.clone();
-
                     let request = script::Request {
                         upgrade,
-                        known_utxos: known_utxos.clone(),
                         cached_ffi_transaction: cached_ffi_transaction.clone(),
                         input_index,
                     };
 
-                    script_verifier.clone().oneshot(request).map_ok(move |rsp| {
-                        utxo_sender.send(rsp).expect("receiver is not dropped");
-                    })
+                    script_verifier.clone().oneshot(request).map_ok(|_r| {})
```

Comment:
Optional: I think with the changes that have been done and by making the `script::Verifier` `Copy`, this could be simplified to:

```suggestion
                    script_verifier.oneshot(request)
```

Reply:
Done in b36577315 / #3446